taxi_service/settings.py

```diff
@@ -140,3 +142,5 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+CRISPY_TEMPLATE_PAC = "bootstrap4"
```

Comment:
There is a typo in the setting `CRISPY_TEMPLATE_PAC`. It should be `CRISPY_TEMPLATE_PACK` to correctly configure the crispy forms package.